client/src/interface/AuthApiData.ts

```diff
@@ -4,9 +4,52 @@ export interface AuthApiDataSuccess {
   message: string;
   user: User;
   token: string;
+  data: CurrentUser;
 }
 
 export interface AuthApiData {
   error?: { message: string };
   success?: AuthApiDataSuccess;
 }
+
+export interface CurrentUser {
```

Comment:
Maybe we can call this CurrentProfile/UserProfile? (to avoid any confusion in future)